src/test/java/duke/task/TaskTest.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TaskTest {
+    @Test
+    void setDone_testTask_false() {
```

Comment:
Is there any reason why Test methods have underscores for naming convention as compared to regular methods?